packages/builder/src/constants/backend/index.js

```diff
@@ -107,10 +107,18 @@ export const FIELDS = {
       },
     },
   },
-  ATTACHMENT: {
+  ATTACHMENT_SINGLE: {
     name: "Attachment",
-    type: FieldType.ATTACHMENT,
-    icon: "Folder",
+    type: FieldType.ATTACHMENT_SINGLE,
+    icon: "Document",
+    constraints: {
+      presence: false,
+    },
+  },
+  ATTACHMENTS: {
+    name: "Attachment List",
+    type: FieldType.ATTACHMENTS,
+    icon: "AppleFiles",
```

Comment:
Is there any reason to use `AppleFiles` for attachment list rather than `Folder`? Probably worth keeping this consistent, incase the `Folder` icon has been used elsewhere for attachments today.

Reply:
This is a current inconsistency. We are showing a different icon when adding the column and in the grid header. I just unified it, using AppleFiles as it's displayed in the grid (hence in the client).
<img width="288" alt="image" src="https://github.com/Budibase/budibase/assets/15987277/0db4abae-f5a9-4ce8-a555-dceb46d6d803">

I am quite sure that this should be unified. Could you confirm @aptkingston ?

Reply:
Ah I see didn't realise it was already inconsistent, nice find! In that case if its now consistent all good!